### DISCUSSION TITLE
Popover not working

### DISCUSSION THREAD

**Discussion by SagarStake** [mdbootstrap/TW-Elements#1438] [Support from community] [answered]
Hi,
I have a page in which I have added a tailwind elements accordion (from the example code) and a tailwind elements popover (from the example code). 
The accordion is working fine as expected where as the popover is not working. 
There are no errors in the console to help me debug. 
I'm clueless as to why it is not working.
You can checkout the code at https://github.com/sagarchand/tailwindPopover to have a look. Please help me fix this popover issue.

Update: I have added a tooltip (which is also not working) and a dropdown to the page (which is working fine)
From what I've noticed, the working version of popover and tooltip inserts a div before the end of </body> tag, which is not happening in my case.

**Comment by Trochonovitz** [comment]
Hi, we saw this problem and [going to fix it](https://github.com/mdbootstrap/Tailwind-Elements/issues/1433), thanks!

**Reply by SagarStake** [reply]
Thank you for the response @Trochonovitz . Do you have an ETA when we can expect the fix to be released? Cheers!

**Reply by Trochonovitz** [reply]
Most probably it will be next week.

**Comment by SergejGleithmann** [comment, marked as answer]
Hey, the tooltip worked for me after adding

```js
var tooltipTriggerList = [].slice.call(
    document.querySelectorAll('[data-te-toggle="tooltip"]')
);
var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
    return new te.Tooltip(tooltipTriggerEl);
});
```
Maybe it helps you too in the meantime?

**Reply by SagarStake** [reply]
Thank you @SergejGleithmann
